pest: add page (#4929)
tldr-pages/tldr: pages/common/pest.md

# pest

> A PHP testing framework with a focus on simplicity.
> More information: <https://pestphp.com>.

> - Initialise a standard Pest configuration in the current directory:

Reviewer: We have pages that say `local path`, I think we need to uniform the expression for `current directory`/`local path`:
```suggestion
- Initialise a standard Pest configuration in the local path:
```

Author: Hmm, personally I feel that current directory makes more sense than local path. 🤔 🤷🏻

Reviewer 2: I agree. `current directory` makes more sense.

`pest --init`

- Run tests in the current directory:

`pest`

- Run tests annotated with the given group:

`pest --group {{name}}`

> - Run tests and generate a coverage report in `stdout`:

Reviewer: I'm not sure about this example description:
```suggestion
- Run tests and print the coverage report to `stdout`:
```

Author: Whoops, yeah... it doesn't make sense. 🤦🏻
